ERROR when building → javax.interceptor depency no longer available at repo.jenkins-ci.org

It's impossible to build from source according to instructions because the dependency **javax.interceptor:javax.interceptor-api:jar:3.1** is no longer present at https://repo.jenkins-ci.org/public/.

```
u54deliv2f@localhost:~/ysoserial$ mvn clean package -DskipTests
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------------< ysoserial:ysoserial >-------------------------
[INFO] Building ysoserial 0.0.6-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for javax.interceptor:javax.interceptor-api:jar:3.1 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.450 s
[INFO] Finished at: 2024-03-09T13:44:05Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project ysoserial: Could not resolve dependencies for project ysoserial:ysoserial:jar:0.0.6-SNAPSHOT: javax.interceptor:javax.interceptor-api:jar:3.1 was not found in https://repo.jenkins-ci.org/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of jenkins has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

Any workaround for a successful build?

--- a/pom.xml
+++ b/pom.xml
@@ -67,6 +67,16 @@
 
 	<repositories>
 		<repository>
+			<id>central</id>
+			<layout>default</layout>
+			<url>https://repo.maven.apache.org/maven2/</url>
+		</repository>		
+		<repository>
+			<id>ysoserial-m2-repo</id>
+			<layout>default</layout>
+			<url>https://raw.githubusercontent.com/frohoff/ysoserial-m2-repo/master</url>
+		</repository>
+		<repository>
 			<id>jenkins</id>
 			<layout>default</layout>
 			<url>https://repo.jenkins-ci.org/public/</url>

